README.md

```diff
@@ -60,26 +60,28 @@ metadata:
 spec:
   llm:
     providers:
+    - type: openai
```

Comment:
It would be better if we also show the new config format for Azure OpenAI provider.
The Azure OpenAI provider is introduced by [this PR](https://github.com/openshift/lightspeed-operator/pull/91).